packages/wonder-blocks-dropdown/src/components/single-select.tsx

```diff
@@ -453,7 +454,7 @@ const SingleSelect = (props: Props) => {
                             onClick={handleClick}
                             disabled={isDisabled}
                             ref={handleOpenerRef}
-                            text={menuText}
+                            content={menuText}
                             opened={open}
```

Comment:
Same question here about applying the aria-label to custom opener!

Reply:
It's in there now, good call!